app/assets/locales/en.json

```diff
@@ -460,6 +465,12 @@
     "resend_activation_link": "If you haven't received an activation email or if you are having a problem using it, click on the button below to request a new activation email.",
     "resend_btn_lbl": "Resend Verification"
   },
+  "confirm_terms_page": {
+    "title": "Confirm terms and conditions",
+    "account_unconfirmed": "Your account has not accepted the terms and conditions.",
+    "message": "Please accept the terms and conditions for Greenlight.",
```

Comment:
It might be better to phrase this a bit more gentle. Also, we don't really refer to the app as Greenlight to the end user since they only know it as BBB.

`To continue using BigBlueButton, please accept our terms and conditions`